custom_model_runner/datarobot_drum/drum/args_parser.py

```diff
@@ -208,6 +208,70 @@ def are_both_labels_present(arg):
                 help="Negative class label for a binary classification case. " + fit_intuit_message,
             )
 
+    @staticmethod
+    def _reg_arg_multiclass_labels(*parsers):
+        class RequiredLength(argparse.Action):
+            ERROR_MESSAGE = "Multiclass classification requires at least 3 labels."
+            MIN_LABELS = 3
+
+            def __call__(self, parser, namespace, values, option_string=None):
+                if len(values) < self.MIN_LABELS:
+                    raise argparse.ArgumentTypeError(self.ERROR_MESSAGE)
+                setattr(namespace, self.dest, values)
+
+        class ParseLabelsFile(argparse.Action):
+            def __call__(self, parser, namespace, values, option_string=None):
+                with open(values) as f:
+                    labels = [label for label in f.read().split(os.linesep) if label]
+                    if len(labels) < RequiredLength.MIN_LABELS:
+                        raise argparse.ArgumentTypeError(RequiredLength.ERROR_MESSAGE)
+                    setattr(namespace, "class_labels", labels)
+
+        def are_labels_double_specified(arg):
+            error_message = (
```

Comment:
move error message inside the `if` below

Reply:
can do in next PR

Reply:
👍